### ISSUE TITLE
Easier way to stop the server

### ISSUE THREAD

**Issue by SpaceboyRoss01** [facekapow/runtime-telnet-server#2]
Can you have the function return module.exports so it could be easier to exit?

**Comment by facekapow** [comment]
Yeah, it's returning nothing right now anyway, so sounds like a good idea. I'll add it in as soon as I can.
